nixos/tests/lemurs.nix

```
@@ -0,0 +1,45 @@
+import ./make-test-python.nix (
+  {
+    pkgs,
+    latestKernel ? false,
+    ...
+  }:
+  {
+    name = "lemurs";
+    meta = with pkgs.lib.maintainers; {
+      maintainers = [
+        stunkymonkey
+        nullcubee
+      ];
+    };
+
+    nodes.machine =
+      { pkgs, lib, ... }:
+      {
+        users.users.alice = {
+          isNormalUser = true;
+          password = "foobar";
+        };
```

This is used in other tests, I figure it would be best to use here as well?
```suggestion
        imports = [ ./common/user-account.nix ];
```